File: Year-2019/Mar/2019-03-12-242/watercow/344.c
class Solution {
public:
	void reverseString(vector<char>& s) {
		for (int i = 0; i < s.size() / 2; i++) {
			char tmp = s[i];
			s[i] = s[s.size() - 1 - i];
			s[s.size() - 1 - i] = tmp;
		}
	}
};

File: Year-2019/Mar/2019-03-12-242/lb/344.java
class Solution {
    public void reverseString(char[] s) {
        char temp;
        for (int i = 0,j =s.length-1;i<j;i++,j--){
            temp = s[i];
            s[i] = s[j];
            s[j] = temp;
        }
    }
}
